data/json/scenarios.json

```diff
@@ -517,19 +517,19 @@
     "points": 3,
     "description": "Despite all the soldiers, guns and minefields, the base you were on got overrun by the dead.  Everyone was ordered to fall back to the armory, but during all the chaos you got lost and you are now stuck in the warehouse all alone.  You are not sure if anyone made it to the armory, or if you are the last man alive.",
     "start_name": "Military Base Warehouse",
-    "allowed_locs": [ "sloc_military_base_warehouse" ],
-    "professions": [ "unemployed", "soldier", "specops", "bio_soldier", "bio_sniper", "labtech", "medic", "mili_pilot" ],
+    "allowed_locs": [ "mil_base_2g" ],
+    "professions": [ "unemployed", "soldier", "specops", "bio_soldier", "bio_sniper", "labtech", "medic" ],
```

Comment:
Outside of scope of PR.

Reply:
I did not make that change. I'm not sure what is causing that honestly. I've had a few of these pop up and I think the best solution is to create a backup of my changes and then sanitize the results after repopulating the master.

Reply:
Unless that has to do with me not appending my changes at the end of the file